ssh.c

```diff
@@ -1695,7 +1695,18 @@ main(int ac, char **av)
 
 	/* Log into the remote system.  Never returns if the login fails. */
 	ssh_login(ssh, &sensitive_data, host, (struct sockaddr *)&hostaddr,
-	    options.port, pw, timeout_ms, cinfo);
+                  options.port, pw, timeout_ms, cinfo);
+
+	if (ssh_packet_connection_is_on_socket(ssh)) {
+		if (ssh_packet_connection_af(ssh) == AF_LOCAL) {
```

Comment:
Just a nit - I think you should use `AF_UNIX` instead of `AF_LOCAL`. IIRC, they are both the same, but `AF_UNIX` is used throughout in the open ssh code.